kura/org.eclipse.kura.core.identity/src/main/java/org/eclipse/kura/core/identity/ValidationUtil.java

```diff
@@ -33,19 +33,30 @@ private ValidationUtil() {
 
     public static void validateNewPassword(final char[] password,
             final PasswordStrengthVerificationService passwordStrengthVerificationService) throws KuraException {
-        if (password.length == 0) {
+        if (password == null || password.length == 0) {
```

Comment:
Would it make sense to call the new validateNewPassword from here to reduce code duplication and possible errors?

Reply:
No because they invoke differents PasswordStrengthVerificationService  methods. I could create a private validateNewPassword method with a null check on identityName to choose which method invoke but it's the same amount of code except for avoiding the duplication of 
```java
requireMaximumLength(NEW_PASSWORD, password, 255)
requireNoWhitespaceCharacters(NEW_PASSWORD, password) 
```

not an unmissable improvement IMHO